src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,329 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: ImageCapture is an API for controlling camera settings and taking photos.
```

Comment:
Good point — changed. It's Image Capture in the spec (when not referring to the object).